cli/azd/pkg/azureutil/resource_group.go

```diff
@@ -38,3 +41,80 @@ func GetResourceGroupsForDeployment(ctx context.Context, azCli tools.AzCli, subs
 
 	return keys, nil
 }
+
+// GetResourceGroupsForEnvironment gets all resources groups for a given environment
+func GetResourceGroupsForEnvironment(ctx context.Context, env *environment.Environment) ([]tools.AzCliResource, error) {
```

Comment:
Can you move these into `pkg/infra/azure_resource_manager.go` ?

Reply:
Which methods?  It wasn't clear to me which ones should go in what file.

Reply:
Discussed offline.  Wallace will move them in his upcoming PR.